contributing-a-devnote.md

```diff
@@ -250,23 +351,95 @@ The first two settings, `jupyter: true` and `format: meca` should not be changed
+
+## Checking your draft
+
+The submission system includes a check suite which allows the DevNotes editors to run up some automated checks on your submission. These checks can be for, things like whether you've added an abstract, the abstract length, or whether all your authors have ORCIDs and whether the DOIs that you're using are valid.
+
+These will be automatically checked when you submit your work and you'll still have chances to correct those afterwards but you can get a head start and check your draft in advance by running the following command.
+
+```sh
+curvenote check bnext-devnotes --kind=dev-notes
+```
+
+And look for potential issues both in the build process and in the Curvenote Checks section.
+
+## Previewing your first draft
+
+If you haven't already, you should go ahead and check a draft of your DevNote as a preview on the DevNote site. You can do this by running the following command.
+
+```sh
+cn submit bnext-devnotes --draft
+```
+
+Check that you are happy with your DevNote in the preview and if not, continue to make changes locally until you are. You can make as many draft submissions as you need to.
+
+Once you're ready we'll carry on and submit your DevNote proper by opening a PR.
+
+## Submitting your DevNote
+
+We're going to submit your DevNote now via a PR and by creating your own fork.
+
+:::{tip} How to provide a good title and description for my PR
+
+This PR is your first interaction with the editorial and review team at the Nucleus Dev Notes site. Please set your title to the title of your DevNote and provide a bit of a introduction your DevNote to the editors and reviewers. If you don't manage to get this in place when you first create the PR, don't worry, you can edit it any time on GitHub.
+
+:::
+
+By now, you will have cloned the repo and made your own folder in the `dev-notes/*` folder, where you've been working so far.
+
+It's important to make sure that any changes that you've made in the repo are only to the contents of your new folder, and nowhere outside of that folder. You can check this by running `git status` and making sure all changes or additions are only in your DevNote folder. If you have changes outside you'll need to revert them before submitting.
+
+Once that's the case then go ahead and just locally commit your changes to the main branch:
+
+```sh
+git add .
+git commit -am "my commit message"
+```
+
+Next we're going to use the GitHub CLI to open a PR and in the process create our own fork of the DevNotes repo in our account. To do this, just run the following command from anywhere within the repository and follow the instructions:
+
+```sh
+gh pr create
+```
```

Comment:
This is where getting people to opt into the GitHub CLI makes things really easy.